backend/src/main/java/service/UserService.java

```diff
@@ -2,22 +2,48 @@
 
 import configuration.DatabaseConnection;
 
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
+//import java.sql.Statement;
```

Comment:
unused import ?

Reply:
Replaced by import java.sql.* (this 2 and more needed).